File: django-ninja/src/apps/cadastro/models/produto.py
# ./src/apps/cadastro/models/produto.py

from django.db import models

class Produto(models.Model):
	descricao = models.CharField(max_length=150)
	ativo = models.BooleanField(default=True)

	def __str__(self):
		return self.descricao

File: front-end-angular/src/app/models/produto.ts
// ./src/app/models/produto.model.ts

export class Produto {
    id: number;
    nome: string;
    descricao: string;
    preco: number;
  
    constructor(id?: number, nome?: string, descricao?: string, preco?: number) {
      this.id = id || 0;
      this.nome = nome || '';
      this.descricao = descricao || '';
      this.preco = preco || 0;
    }
  }
  